# no inputs
---
float64[7] q
